Dockerfile.dapper

```diff
@@ -0,0 +1,22 @@
+FROM debian
+
+ARG DAPPER_HOST_ARCH
+ENV HOST_ARCH=${DAPPER_HOST_ARCH} ARCH=${DAPPER_HOST_ARCH}
+ENV CATTLE_HELM_VERSION v2.10.0-rancher4
```

Comment:
Not sure why this is here.  Looks like we're pulling latest from kubernetes/helm/master